portal/server/utils/csv-utils.js

```diff
@@ -111,15 +111,15 @@ const parseXlsxToCsvArrays = (worksheet) => {
 
     if (!firstRow) {
       // If the row has no data in the final columns of the row, we need to fill in the empty cells with
-      // empty strings and cell addresses so that the csv parser library can parse the data correctly
+      // null values and cell addresses so that the csv parser library can parse the data correctly
       while (columnCount < headerCount) {
-        rowData.push('');
+        rowData.push(`${null}`);
```

Comment:
so when this gets parsed, is it once it gets to JSON from the `csv-parser` that this then gets parsed as 'null' in the json data?

Reply:
Ahh looked into where this is used more and I hadn't tested the csv download and it gives me spreadsheet with lots of strings of the word null in it which we don't want. 

It's the with cell addresses where using null in the string matches the empty cells not at the end of the row, but given I have now added handling to the mapping for empty strings since we need it for csv uploads anyway I will simply revert this part back.